routes/web.php

```diff
@@ -41,3 +41,6 @@
 Route::post('/admin/word/update', 'WordController@update');
 Route::get('/admin/word/delete/{word}', 'WordController@delete');
 
+//User RD functions for Admin
+Route::get('/admin/user', 'UserController@index');
+Route::get('/admin/user/delete/{user}', 'UserController@delete');
```

Comment:
delete method should not be get request, it must be delete request.

note: remember the HTTP verbs which are GET, POST, PUT, PATCH and DELETE

Reply:
Okay, I understand. And It's not only about this route. I'll fix all delete route.